Pandora/GaudiPandora/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 
 # Modules
-gaudi_add_module(GaudiPandora
+gaudi_add_module(k4GaudiPandora
```

Comment:
As the module name is renamed, maybe we could also rename the directory name.

Reply:
Hi @mirguest 

I have renamed the directory name, please check it.

Best,
Wenxing